test/integration/cluster/test_cluster_promote.py

```diff
@@ -246,43 +453,85 @@ def test_cluster_promote_bad_auth(tt_cmd, etcd, tmpdir_with_cfg):
     tmpdir = tmpdir_with_cfg
-    etcdcli = etcd.conn()
+    conn = instance.conn()
     key = to_etcd_key("all")
-    etcdcli.put(key, test_auth_cfg_init)
+    if instance_name == "etcd":
+        conn.put(key, test_auth_cfg_init)
+    else:
+        conn.insert(space_name="config_storage", values=[key, test_auth_cfg_init, 2])
     try:
-        etcd.enable_auth()
+        if instance_name == "etcd":
+            instance.enable_auth()
 
         if auth == "url":
             env = None
             url = (
-                f"http://{DEFAULT_ETCD_USERNAME}:{DEFAULT_ETCD_PASSWORD}@"
-                f"{etcd.host}:{etcd.port}/prefix?timeout=5"
+                f"http://{instance.connection_username}:{instance.connection_password}@"
+                f"{instance.host}:{instance.port}/prefix?timeout=5"
             )
             promote_cmd = [tt_cmd, "cluster", "rs", "promote", "-f", url, "instance-002"]
         elif auth == "flag":
             env = None
-            url = f"{etcd.endpoint}/prefix?timeout=5"
+            url = f"{instance.endpoint}/prefix?timeout=5"
             promote_cmd = [tt_cmd, "cluster", "rs", "promote", "-f",
-                           "-u", DEFAULT_ETCD_USERNAME,
-                           "-p", DEFAULT_ETCD_PASSWORD,
+                           "-u", instance.connection_username,
+                           "-p", instance.connection_password,
                            url, "instance-002"]
         elif auth == "env":
-            env = {"TT_CLI_ETCD_USERNAME": DEFAULT_ETCD_USERNAME,
-                   "TT_CLI_ETCD_PASSWORD": DEFAULT_ETCD_PASSWORD}
-            url = f"{etcd.endpoint}/prefix?timeout=5"
+            env = {
+                (
+                    "TT_CLI_ETCD_USERNAME"
+                    if instance_name == "etcd"
+                    else "TT_CLI_USERNAME"
+                ): instance.connection_username,
+                (
+                    "TT_CLI_ETCD_PASSWORD"
+                    if instance_name == "etcd"
+                    else "TT_CLI_PASSWORD"
+                ): instance.connection_password,
+            }
+            url = f"{instance.endpoint}/prefix?timeout=5"
             promote_cmd = [tt_cmd, "cluster", "rs", "promote", "-f", url, "instance-002"]
 
         rc, out = run_command_and_get_output(promote_cmd, cwd=tmpdir, env=env)
         assert rc == 0
         assert f'Patching the config by the key: "{key}"' in out
 
-        etcd.disable_auth()
-        etcdcli = etcd.conn()
-        actual, _ = etcdcli.get(to_etcd_key("all"))
-        assert test_auth_cfg_expected == actual.decode("utf-8")
+        if instance_name == "etcd":
+            instance.disable_auth()
+        conn = instance.conn()
+
+        content = ""
+        if instance_name == "etcd":
+            content, _ = conn.get(to_etcd_key("all"))
+            content = content.decode("utf-8")
+        else:
+            content = conn.select(space_name="config_storage", key=key)
```

Comment:
Nit: we should use tcs API calls, the internal implementation could be changed in the future:
https://github.com/tarantool/tarantool/tree/master/doc/examples/config/config-storage#get
```suggestion
            content = conn.call("config.storage.get", {key})
```

Reply:
I did the same requests for `replicaset publish/show` tests. Should I change it?

Reply:
Yes, I have missed that. It's better to do it in a separate commit.

Reply:
Corrected on tcs API calls for all tests.